blob/azureblob/azureblob.go

```diff
@@ -26,6 +26,11 @@
 // The default URL opener will use credentials from the environment variables
 // AZURE_STORAGE_ACCOUNT, AZURE_STORAGE_KEY, and AZURE_STORAGE_SAS_TOKEN.
 // AZURE_STORAGE_ACCOUNT is required, along with one of the other two.
+// AZURE_BLOB_DOMAIN can optionally be used to provide an Azure Environment
```

Comment:
Naming nit: let's use `AZURE_STORAGE_` prefix for consistency with the other ones.